crates/apub/src/fetcher/webfinger.rs

```diff
@@ -84,7 +84,9 @@ where
 
   *request_counter += 1;
   if *request_counter > context.settings().http_fetch_retry_limit {
-    return Err(LemmyError::from(anyhow!("Request retry limit reached")));
+    return Err(LemmyError::from(anyhow::anyhow!(
```

Comment:
this is a bugfix

Reply:
This can just be a `LemmyError::from_message`.